src/components/form/form.ts

```diff
@@ -97,55 +93,55 @@ export class MForm extends ModulVue {
         this.formGroup.reset();
     }
 
-    private _getAllFormErrors(formGroup: FormGroup | FormArray): ControlError[] {
-        let result: ControlError[] = formGroup.errors;
+    private _getAllFormErrors(control: FormGroup | FormArray): ControlError[] {
+        let result: ControlError[] = control.errors;
 
-        formGroup.controls.forEach(c => {
+        control.controls.forEach(c => {
             if (c instanceof FormControl) {
                 result = result.concat(c.errors);
-            } else {
-                result = result.concat(this._getAllFormErrors(c as FormGroup | FormArray));
+            } else if (c instanceof FormGroup || c instanceof FormArray) {
+                result = result.concat(this._getAllFormErrors(c));
             }
```

Comment:
pas certain de l'implémentation exacte, mais on pourrait faire un méthode abstract dans le parent AbstractControl genre
`.combineErrors(combinedErrors: ControlError[]) {
    combinedErrors.concat(this.errors)
    this.controls.forEach( c=> c.combineErrors(combinedErrors));
}`
et simplement caller control.combineErrors(result);
et chaque type de control call ses controls avec la même méthode et selon le type il fait

Reply:
je vais regarder ça oui!